### PR TITLE
Modified to allow arbitrary metadata to show up in RV.

### PR DESCRIPTION
**Pull request by richardssam**
MKV in particular allows arbitrary key/value metadata pairs. The previous code only allowed a known list of parameters to be passed through to RV. This version has a list of parameters that should NOT be passed through.

### Describe what you have tested and on which operating system.
I've tested loading mov and mkv files on windows.

Below is an example of getting a field called "samfield" to show up in the info display.

image

This metadata was added using the following ffmpeg command:
```
ffmpeg -i chimera_cars_srgb-test_mov-mjpeg.mov -metadata samfield=samtest -c copy chimera_cars_srgb-test_mov-mjpeg.mkv
```

### CODE REVIEW

File: src/lib/image/MovieFFMpeg/MovieFFMpeg.cpp [hunk 1]
```diff
@@ -832,6 +838,17 @@ isMetadataField(string check)
     return false;
 }
 
+
+bool
+ignoreMetadataField(string check)
```
Comment (bernie-laberge) [inline src/lib/image/MovieFFMpeg/MovieFFMpeg.cpp:843]: Minor detail : could be passed by reference (string&  check) 
That being said, I know it was exactly like that before ! :-)

**Review by bernie-laberge — APPROVED** [review]
Looks good to me. 
Thank you ! 
Your contribution is very much appreciated !

File: src/lib/image/MovieFFMpeg/MovieFFMpeg.cpp (hunk 1 above)
Comment (richardssam) [reply src/lib/image/MovieFFMpeg/MovieFFMpeg.cpp:843]: No reason why not, I'll change it tomorrow AM.

Comment (richardssam) [reply src/lib/image/MovieFFMpeg/MovieFFMpeg.cpp]: Should be addressed.

File: src/lib/image/MovieFFMpeg/MovieFFMpeg.cpp [hunk 2]
```diff
@@ -5467,26 +5537,19 @@ MovieFFMpegIO::MovieFFMpegIO(CodecFilterFunction codecFilter,
...
-    //     {
-    //         int poolSize = atoi(c);
-    //     }
-    //     if (poolSize > 0) globalContextPool = new ContextPool(poolSize);
-    //     else
-    //     {
-    //         report("Disabling mio_ffmpeg context thread pool.", true);
-    //     }
-    // }
+    if (!globalContextPool)
```
Comment (bernie-laberge) [inline src/lib/image/MovieFFMpeg/MovieFFMpeg.cpp:5540]: I believe this is a merge issue here.
This FFmpeg context pool should be commented out as it no longer works with FFmpeg 6+.

File: src/lib/image/MovieFFMpeg/MovieFFMpeg.cpp (hunk 1 above)
Comment (bernie-laberge) [reply src/lib/image/MovieFFMpeg/MovieFFMpeg.cpp]: Thanks !

**Comment by richardssam** [comment]
We can close this, it should be a duplicate of #504, but with a clean commit.

**Pull request closed without merging**